docs/networking.adoc

```diff
@@ -0,0 +1,766 @@
+[source,console]
+----
+root@oxz_nexus_c4c40c19-60de-4c8b-b201-2a367d8aa322:~# ipadm
+ADDROBJ           TYPE     STATE        ADDR
+lo0/v4            static   ok           127.0.0.1/8
+vopte0/public     dhcp     ok           172.30.2.6/32
+lo0/v6            static   ok           ::1/128
+oxControlService12/ll addrconf ok       fe80::8:20ff:fe57:a9%oxControlService12/10
+oxControlService12/omicron6 static ok   fd00:1122:3344:104::4/64
+----
+
+External connectivity uses the (aptly-named) `vopte0/public` address.  But the address there is 172.30.2.6, not in the configured range of 172.20.26.1 - 172.26.1.10.  It's a coincidence that these ranges look so similar.  No matter the actual customer-provided addresses, Nexus always uses fixed 172.30 prefixes for DNS, Nexus, and Boundary NTP.  Customers can choose what range is used for their VPCs.
+
+Still, one of the real external addresses must map to this Nexus zone.  How do we find out what that is?  We can use `opteadm` in the GZ:
+
+[source,console]
+----
+BRM44220005 # /opt/oxide/opte/bin/opteadm list-ports
+LINK                             MAC ADDRESS              IPv4 ADDRESS     EXTERNAL IPv4    IPv6 ADDRESS                             EXTERNAL IPv6                            STATE
+opte0                            A8:40:25:FF:A7:C1        172.30.2.6       172.20.26.4      None                                     None                                     running
+opte1                            A8:40:25:F2:6F:96        172.30.0.5       172.20.26.12     None                                     None                                     running
+opte2                            A8:40:25:FE:2F:F1        172.30.0.25      172.20.26.24     None                                     None                                     running
+opte3                            A8:40:25:F5:B0:49        172.30.0.37      172.20.26.34     None                                     None                                     running
+opte4                            A8:40:25:F2:52:4B        172.30.0.9       172.20.26.35     None                                     None                                     running
+opte5                            A8:40:25:F7:F7:4A        172.30.0.39      172.20.26.37     None                                     None                                     running
+opte6                            A8:40:25:F7:19:71        172.30.0.41      172.20.26.39     None                                     None                                     running
+opte7                            A8:40:25:F2:BB:4B        192.168.0.17     172.20.26.45     None                                     None                                     running
+----
+
+Recall that OPTE is part of boundary services.  This command lists key configuration: it says that external IP 172.20.26.4 maps to an internal IPv4 address 172.30.2.6 on `opte0`.  This corresponds with `vopte0` in the Nexus zone (because it's the same-numbered device name).
```

Comment:
This is the first mention of OPTE as a named component. Should we define it earlier or at least spell out that it's the Oxide Packet Transformation Engine?

I'm not sure I'd characterize OPTE as entirely part of boundary services. I think of it as its own component, because it's also here for all VM-VM communication. Dunno.

Reply:
OPTE does show up earlier, in the bullet list at the start of this section.

I didn't mean to imply that OPTE was entirely within boundary services.  I'll tweak this to hopefully make that clearer.